src/OpenTelemetry.Exporter.Console/ConsoleExporterHelperExtensions.cs

```diff
@@ -19,26 +19,24 @@
 
 namespace OpenTelemetry.Trace
 {
-    public static class TracerProviderBuilderExtensions
+    public static class ConsoleExporterHelperExtensions
     {
         /// <summary>
         /// Registers a ConsoleActivity exporter to a processing pipeline.
         /// </summary>
         /// <param name="builder"><see cref="TracerProviderBuilder"/> builder to use.</param>
         /// <param name="configure">Exporter configuration options.</param>
         /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
-        public static TracerProviderBuilder UseConsoleExporter(this TracerProviderBuilder builder, Action<ConsoleExporterOptions> configure = null)
+        public static TracerProviderBuilder AddConsoleExporter(this TracerProviderBuilder builder, Action<ConsoleExporterOptions> configure = null)
         {
-            // TODO: Rename to AddConsoleExporter?
             if (builder == null)
             {
                 throw new ArgumentNullException(nameof(builder));
             }
 
-            var exporterOptions = new ConsoleExporterOptions();
-            configure?.Invoke(exporterOptions);
-            var consoleExporter = new ConsoleExporter(exporterOptions);
-            return builder.AddProcessor(new SimpleActivityProcessor(consoleExporter));
+            var options = new ConsoleExporterOptions();
+            configure?.Invoke(options);
+            return builder.AddProcessor(new SimpleActivityProcessor(new ConsoleExporter(options)));
```

Comment:
Do we let user pick Simple vs Batched for other exporters?

Reply:
I guess it doesn't make much sense for console exporter to be batched as it is specifically designed for local dev scenario?

Reply:
Yea. I was asking general approach for other exporters which can operate in simple and batch mode..